config/lib/mongoose.js

```diff
@@ -24,25 +25,28 @@ module.exports.connect = function (cb) {
 
```

Comment:
Remove `var _this = this;` as it's never used (weird how eslint didn't warn about this?)